dist/windows/helper.nsi

```diff
@@ -0,0 +1,61 @@
+; Code taken from https://nsis.sourceforge.io/VersionCompleteXXXX
```

Comment:
I think it should have a better file name. for example `versionComplete.nsi`.
And maybe put them to a separate folder `3rdparty` to signify we don't modify them and only sync to upstream.